### DISCUSSION TITLE
No timestamp when a track was favorited (Favorite tracks)?

### DISCUSSION THREAD

**Discussion by mrdc** [MarshalX/yandex-music-api#498] [Q&A] [answered]
Hello,

I'm checking Yandex.Music API and it looks like they don't store a timestamp when I favorited a track (Favorite tracks playlist). 
But at the same time I see the timestamp in All tracks playlist: in mobile apps tracks are divided: Yesterday, This Week, Last Week, Junuary etc. Or there is a possibility to retrieve timestamps?

**Comment by MarshalX** [comment, marked as answer]
Hello. 

<img width="522" alt="image" src="https://user-images.githubusercontent.com/15520314/149508186-a13cac70-e639-46df-89d2-27df34420391.png">

where the `l` is `c.users_likes_tracks()` where is `c` is instance of `Client`
